discord/channel.py

```diff
@@ -3019,13 +3019,13 @@ def permissions_for(self, obj: Any = None, /) -> Permissions:
         Parameters
         -----------
         obj: :class:`User`
-            The user to check permissions for. This parameter is ignored
-            but kept for compatibility with other ``permissions_for`` methods.
+            The user to check permissions for.
 
         Returns
         --------
         :class:`Permissions`
-            The resolved permissions.
+            The user to check permissions for. This parameter is ignored
+            but kept for compatibility with other ``permissions_for`` methods.
```

Comment:
Why did you move this..?

Reply:
My bad, removed it when I implemented this for dms and re-added it at the wrong place